presto-docs/src/main/sphinx/release/release-0.236.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.236
+=============
+
+General Changes
+_______________
+* Fix query failure due to searched case expression with large number of cases.
+* Improve experience by warning user of poorly performant ``UNION DISTINCT`` usage.
+
+Verifier Changes
+________________
+* Add ``ErrorType`` of query failures to verification outputs.
+* Add support to auto-resolve result mismatch for structured-type columns.
+* Add support to auto-resolve result mismatch in case the query uses functions to be ignored.
+
+Hive Changes
+____________
+* Fix `ANALYZE` query failure for tables with structure-typed columns. (:issue:`14494`).
+* Fix Parquet schema mismatch when type is upgraded from ``int32`` to ``int64``.
+* Improve planning time for queries that scan large number of partitions across multiple partition columns.
+* Add local data caching support with `Alluxio <https://www.alluxio.io/>`_ cache library. To enable it, set configuration property ``cache.enabled=true`` and ``cache.type=ALLUXIO``.
```

Comment:
Going with the language in https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#phrasing - lets rephrase the second sentence as:

```
This can be enabled by setting the configuration property ``cache.enabled`` to ``true`` and ``cache.type`` to `ALLUXIO``
```